packages/valory/services/trader_pearl/service.yaml

```diff
@@ -74,15 +73,22 @@ models:
       slippage: 0.01
       store_path: ./data
```

Comment:
can we have this like `${STORE_PATH:str:/data/}`? and others also of the same format so that they can be overridable from the environment variables?

Reply:
Yes I don't see any issue with making this change.